test/inspector/test-inspector-stops-no-file.js

```diff
@@ -0,0 +1,18 @@
+'use strict';
+const common = require('../common');
+const spawn = require('child_process').spawn;
+
+let child = null;
+
+function DidNotShutdown() {
+  if (child)
+    child.kill();
+  common.fail('Did not shut down');
+}
+
+const timeout = setTimeout(DidNotShutdown, 60 * 1000);
```

Comment:
I doubt it will ever trigger during a normal run because the test runner is going to kill the test after 5 or 10 seconds.  It's probably easiest to simply scrap this code.